src/classes/COUR_CannotDelete_TEST.cls

```diff
@@ -0,0 +1,390 @@
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2020
+* @group Course
+* @group-content ../../ApexDocContent/Course.htm
+* @description Tests for COUR_CannotDelete_TDTM
+*/
+@isTest
+public class COUR_CannotDelete_TEST {
+    /*********************************************************************************************************
+    * @description Retrieves the Administrative record type Id. 
+    */
+    public static String adminAccRecTypeId = UTIL_Describe.getAdminAccRecTypeID(); 
+    
+    /*********************************************************************************************************
+    * @description Test method to test if Prevent_Course_Deletion__c is enabled in Hierarchy Settings, and
+    * Course has a Course Offering record associated to it, then it cannot be deleted.
+    */
+    @isTest
+    public static void courCannotDeleteWithCourOffering() {
+        UTIL_CustomSettingsFacade.getSettingsForTests(new Hierarchy_Settings__c
+                                                            (Account_Processor__c = adminAccRecTypeId,
+                                                            Prevent_Course_Deletion__c = True));
+        
+        //We're using "Administrative" record type for this new Account since
+        //we're not ready (we need to determine how best to handle retrieving recordTypeIds that are not
+        //in EDA Settings) to create a new method to retrieve the Universtiy_Department Account
+        //record type in UTIL_Describe. 
+        List<Account> accounts = UTIL_UnitTestData_TEST.getMultipleTestAccounts(1, adminAccRecTypeId); 
```

Comment:
Please add the class name to adminAccRecTypeId here and in other places in the test class so we are explicit about referencing a static variable.